wger/weight/templates/overview.html

```diff
@@ -110,6 +110,12 @@
                 {% trans "Import from spreadsheet" %}
             </a>
         </li>
+        <li>
+            <a href="https://www.fitbit.com/oauth2/authorize?response_type=token&client_id=228LLX&redirect_uri=http%3A%2F%2F127.0.0.1%3A8000%2Fen%2Fweight%2Ffitbit&scope=activity%20heartrate%20location%20nutrition%20profile%20settings%20sleep%20social%20weight&expires_in=604800" {% auto_link_css flavour %}>
```

Comment:
I think this url should be set in the backend and added to the template context. 
Also this redirect url won't work after hosting.

Reply:
We meant to test the weight imports with localhost first before changing the url to point to the deployed version.

Reply:
Done.